README.md

```diff
@@ -44,18 +44,17 @@ Once installed, you can access the example programs from the menu :
 
 ```File > Examples > ChirpSDK > Example ```
 
-and you can include the headers to use Chirp in your own code by using :
+and you can include the headers to use Chirp in your own code by adding :
 
-```Sketch > Import Library > ChirpSDK```
+```#include "chirp_connect.h"```
```

Comment:
Don't you need to add the library anyway ?

Reply:
Nope Arduino locates it automatically somehow. In fact when you use the `Import Library`, all it does is include all of the unnecessary chirp header files.

Reply:
Worth double checking though

Reply:
👍 👍 👍 👍